src/main/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/JavadocCommentsLexer.g4

```diff
@@ -159,7 +159,7 @@ HtmlComment_TEXT
 // Example: " * This is a line in a Javadoc block"
 mode START_OF_LINE;
 StartOfLine_LEADING_ASTERISK: [ \t]* '*' -> channel(LEADING_ASTERISKS), popMode, type(LEADING_ASTERISK);
-POP_MODE: . {_input.seek(_input.index() - 1);} -> skip, popMode;
+POP_MODE: {true}? -> skip, popMode;
```

Comment:
This predicate is nonsense. Why do we need this?

Reply:
Why not just pop from the node stack twice in the `StartOfLine_LEADING_ASTERISK` rule then?

Reply:
Or just match one of any token and do `more’?